### DISCUSSION TITLE
speedster

### DISCUSSION THREAD

**Discussion by qwerkilo** [Amblyopius/Stable-Diffusion-ONNX-FP16#14] [Ideas]
I don't know if the speedster has any effect on the speed of sd image generation,link: https://github.com/nebuly-ai/nebullvm/tree/main/apps/accelerate/speedster ,It has an optimized introduction to the onnx model

**Comment by Amblyopius** [comment]
At first sight it may lack the maturity needed to outperform what we currently do but it's interesting from the perspective that we could contribute back to them and eventually get a benefit from the combined work of us and others. I'll have a look at it but my expectation is that their ONNX part is still in its first steps.
